### DISCUSSION TITLE
import-scan/reimport-scan

### DISCUSSION THREAD

**Discussion by RKCP** [DefectDojo/django-DefectDojo#6008] [Q&A]
Hi,

I may be using the API wrong, but no matter what payload I send to the two endpoints in the title, I always get `"product_name parameter missing"` as a response, along with `400 Bad Request`.

If I execute these endpoints through the swagger docs, there is no issue, but in Postman I keep getting this response. I even duplicated the swagger doc payload and still get the issue.

This happens whether I run defectdojo locally, or use the demo tool.

I have attached an image of a request I am sending. Any one have ideas on what is going wrong?

<img width="859" alt="image" src="https://user-images.githubusercontent.com/43743193/157442477-e0f2c4f2-9e69-4e86-b531-2914ee3f7cb0.png">

**Comment by thmslmtr** [comment]
Are there any solutions in the meantime?

**Comment by romulogm** [comment]
Try passing parameters on form-data.

**Comment by ctrung** [comment]
@romulogm nailed  it. 

To be 100% safe, one can import the API contract (eg. https://demo.defectdojo.org/api/v2/doc/?format=openapi) into Postman and have a look at how the requests are constructed.

**Comment by reichertan** [comment]
What helped me was adding a "/" to the end of the path. So, base-url/api/v2/scan-import/
